templates/claim_payment.html

```diff
@@ -22,8 +22,8 @@ <h6>You can claim a payment of {{payment.amount|int2asset}} {{ config["ASSET_NAM
                                     {{ qrcode_svg | safe }}
                                     <br/>
                                     <p>Scan the QR code or click the link to claim your {{config["ASSET_NAME"]}}.</p>
-                                    <br/>
                                 </a>
+				<p>{{ config["ASSET_NAME"] }} App is available on <a href={{ config["APPLE_APP_STORE_URL"] }}>Apple App Store</a> and <a href={{ config["GOOGLE_PLAY_STORE_URL"] }}>Google Play Store</a></p>
```

Comment:
move below the submit button and replace the text with two icon buttons similar to the app download buttons on https://zap.me/

Reply:
i've got something uploading in the tnext PR.

Reply:
which PR?

Reply:
the buttons: should be deployed now. i pushed like an hour ago?

Reply:
PR is a "pull request", you mean you are pushing/deploying the new changes to caprover

Reply:
I want you to use the icons like on zap.me 
![image](https://user-images.githubusercontent.com/42459/116194613-2cdb7180-a785-11eb-8fc3-69be15fe481c.png)

Reply:
need to figure out how to use css and do the same as the zap.me buttons.

Reply:
Its pretty simple if you look at the source code

```html
<!-- SVG Sprite-->
<div class="svg-icon" style="display: none;">
    <svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
         <symbol id="icon-google"><path fill="currentColor" d="M8.2 4.912a1.671 1.671 0 00-1.603-.077l8.65 8.67 2.828-2.842L8.2 4.913zm-2.398.726c-.11.23-.168.488-.168.752v15.858c0 .265.059.519.17.745l8.646-8.688-8.648-8.667zm9.446 9.466l-8.636 8.678c.229.106.473.152.716.152.3 0 .6-.072.872-.235l9.877-5.76-2.829-2.835zm6.588-2.251l-.01-.006-2.742-1.597-3.04 3.054 3.042 3.047 2.75-1.603c.51-.31.815-.85.815-1.448a1.68 1.68 0 00-.815-1.447z"/></symbol>
    </svg>
</div>

...

<!-- Button -->
<a class="bttn bttn--icon-left" href="https://play.google.com/store/apps/details?id=me.zap.zapapp" target="_blank">
    <span class="bttn__icon">
        <svg class="icon icon-google" width="28" height="28" viewBox="0 0 28 28">
	    <use xlink:href="#icon-google"></use>
        </svg>
    </span>
    <span class="bttn__text">Download on google play</span>
</a>
```

Reply:
i added those lines above and got the following:
![image](https://user-images.githubusercontent.com/47799652/116324686-aa01f780-a814-11eb-98e4-41288c195ad6.png)

so it looks like it only added the sprite but missing the button.

Reply:
that is because you do not have any CSS for the "bttn__icon" class.

I dont want you to copy and paste exactly, I want you to adapt the concept to your buttons

Reply:
Also change your button classes to `class="btn btn-secondary btn-sm"`

Reply:
try this

```html
<div class="svg-icon" style="display: none;">
    <svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
         <symbol id="icon-google"><path fill="currentColor" d="M8.2 4.912a1.671 1.671 0 00-1.603-.077l8.65 8.67 2.828-2.842L8.2 4.913zm-2.398.726c-.11.23-.168.488-.168.752v15.858c0 .265.059.519.17.745l8.646-8.688-8.648-8.667zm9.446 9.466l-8.636 8.678c.229.106.473.152.716.152.3 0 .6-.072.872-.235l9.877-5.76-2.829-2.835zm6.588-2.251l-.01-.006-2.742-1.597-3.04 3.054 3.042 3.047 2.75-1.603c.51-.31.815-.85.815-1.448a1.68 1.68 0 00-.815-1.447z"/></symbol>
    </svg>
</div>


...


<a href="https://play.google.com/store/apps/details?id=me.zap.zapapp" target="_blank" class="btn btn-outline-secondary btn-sm">
    <svg viewBox="0 0 28 28" style="vertical-align: bottom" width="20" height="20">
         <use xlink:href="#icon-google"></use>
    </svg>
    Download on google play
</a>
```

Reply:
this is what i have now:
![image](https://user-images.githubusercontent.com/47799652/116329988-a70d0400-a820-11eb-9aeb-8ef65b4aeaeb.png)


the one with the image in the button.

Reply:
change it like this:
![image](https://user-images.githubusercontent.com/42459/116330652-12a3a100-a822-11eb-9a9d-b5985582c9cd.png)

Reply:
This is what i have:
![image](https://user-images.githubusercontent.com/47799652/116330736-44b50300-a822-11eb-9fc0-3df3f9c90e78.png)

Reply:
oh you want me to delete the Zero App is available.